planning/motion_velocity_planner/autoware_motion_velocity_boundary_departure_prevention_module/src/boundary_departure_prevention_module.cpp

```diff
@@ -656,6 +730,30 @@ std::unordered_map<DepartureType, bool> BoundaryDeparturePreventionModule::get_d
 
   return diag;
 }
+
+bool BoundaryDeparturePreventionModule::is_departing_continuously()
+{
+  if (!last_no_critical_dpt_time_ptr_) {
+    last_no_critical_dpt_time_ptr_ = std::make_unique<double>(clock_ptr_->now().seconds());
+    return false;
+  }
+
+  const auto is_found =
```

Comment:
```suggestion
  const auto is_critical_departure_found =
```

Reply:
updated in https://github.com/autowarefoundation/autoware_universe/pull/11126/commits/5195bee4fe3ac08811040960a5cf0e99553db434